omod/src/main/java/org/openmrs/module/metadatasharing/web/bean/ConfigureForm.java

```diff
@@ -13,8 +13,6 @@
  */
 package org.openmrs.module.metadatasharing.web.bean;
 
-import org.openmrs.module.metadatasharing.web.controller.ConfigureController;
-
```

Comment:
Do we need this change as part of the PR?

Reply:
i have compiled and deployed it in a RA  and its working fine. Other changes were due to other facts of unit testing which am going to reserve then in second ticket META-372,but everything is working fine with these changes

Reply:
That import is uneccessary , it has not any impact on other files, but i can put it back